Cargo.toml

```diff
@@ -0,0 +1,5 @@
+[workspace]
+members = [
+    "buildpacks/ruby",
+    "commons"
```

Comment:
I called this `"common"` in the Node.js buildpack. I'm not sure either is more correct, but maybe we could be consistent?

Reply:
I used "s" on other directories like `layers` and `steps`. It looks like some of the nodejs buildpack dirs do too.